Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM golang:1.18-rc-alpine3.14
+
+RUN mkdir /platform-go-challenge
+COPY . /platform-go-challenge
+WORKDIR /platform-go-challenge
+RUN go build cmd/platform-go-challenge/main.go
```

Comment:
Would be possible to do something here to improve the performance of the final app image?

Reply:
The only thing I can think of at the moment is to have a more lightweight base image from the start which is why I picked this one.
I am weak in this kind of stuff and I would love to learn more about them and gain hands on experience.